### PR TITLE
Fix: centos8 reached EOL and ovn 21.03 tarball unmatch with ovs 2.15 tarball issue

### LINKED ISSUE #3545: make build-dpdk failed since the centos8 reached EOL and ovn 21.03 tarball unmatch with ovs 2.15 tarball
# Bug Report

## Expected Behavior

build out dpdk based ovn and ovs

## Actual Behavior

**1:**

```
docker buildx build --platform linux/amd64 -t kubeovn/kube-ovn-dpdk:19.11-v1.13.0 -o type=docker -f dist/images/Dockerfile.dpdk1911 dist/images/
[+] Building 54.6s (7/16)
 => [internal] load build definition from Dockerfile.dpdk1911                                                                                                                                                         0.0s
 => => transferring dockerfile: 2.50kB                                                                                                                                                                                0.0s
 => [internal] load .dockerignore                                                                                                                                                                                     0.0s
 => => transferring context: 143B                                                                                                                                                                                     0.0s
 => [internal] load metadata for docker.io/library/centos:8                                                                                                                                                           3.0s
 => [auth] library/centos:pull token for registry-1.docker.io                                                                                                                                                         0.0s
 => [dpdk 1/3] FROM docker.io/library/centos:8@sha256:a27fd8080b517143cbbbab9dfb7c8571c40d67d534bbdee55bd6c473f432b177                                                                                               46.2s
 => => resolve docker.io/library/centos:8@sha256:a27fd8080b517143cbbbab9dfb7c8571c40d67d534bbdee55bd6c473f432b177                                                                                                     0.0s
 => => sha256:a27fd8080b517143cbbbab9dfb7c8571c40d67d534bbdee55bd6c473f432b177 762B / 762B                                                                                                                            0.0s
 => => sha256:a1801b843b1bfaf77c501e7a6d3f709401a1e0c83863037fa3aab063a7fdb9dc 529B / 529B                                                                                                                            0.0s
 => => sha256:5d0da3dc976460b72c77d94c8a1ad043720b0416bfc16c52c45d4847e53fadb6 2.14kB / 2.14kB                                                                                                                        0.0s
 => => sha256:a1d0c75327776413fa0db9ed3adcdbadedc95a662eb1d360dad82bb913f8a1d1 83.52MB / 83.52MB                                                                                                                     40.0s
 => => extracting sha256:a1d0c75327776413fa0db9ed3adcdbadedc95a662eb1d360dad82bb913f8a1d1                                                                                                                             4.3s
 => [internal] load build context                                                                                                                                                                                     0.0s
 => => transferring context: 6.93kB                                                                                                                                                                                   0.0s
 => ERROR [dpdk 2/3] RUN yum -y install dpdk-devel &&   dnf install -y --enablerepo=powertools   gcc make numactl-devel meson   unbound nc iptables ipset hostname &&   cd /usr/src/ &&   curl http://fast.dpdk.org/  5.2s
------
 > [dpdk 2/3] RUN yum -y install dpdk-devel &&   dnf install -y --enablerepo=powertools   gcc make numactl-devel meson   unbound nc iptables ipset hostname &&   cd /usr/src/ &&   curl http://fast.dpdk.org/rel/dpdk-20.11.tar.gz | tar xz &&   cd dpdk-20.11 &&   meson builddir && cd builddir &&   meson configure -Dapps='pdump proc-info' -Dexamples='' -Dtests=false -Denable_kmods=false -Denable_docs=false &&   ninja && ninja install &&   dnf remove -y make meson gcc cpp &&   dnf clean all:
#0 1.550 CentOS Linux 8 - AppStream                      102  B/s |  38  B     00:00
#0 1.555 Error: Failed to download metadata for repo 'appstream': Cannot prepare internal mirrorlist: No URLs in mirrorlist
```

It's because [ CentOS Linux 8 will reach End Of Life (EOL) on December 31st, 2021. ](https://www.centos.org/centos-linux-eol/)

**2.**

```
utilities/ovn-trace.c: In function 'parse_lb_option':
utilities/ovn-trace.c:217:10: error: too few arguments to function 'inet_parse_active'
  217 |     if (!inet_parse_active(s, 0, &ss, false)) {
      |          ^~~~~~~~~~~~~~~~~
In file included from /home/larena/alpine_build/shared/home/larena/aports/testing/ovn/src/openvswitch-2.13.7/lib/stream.h:25,
                 from utilities/ovn-trace.c:46:
/home/larena/alpine_build/shared/home/larena/aports/testing/ovn/src/openvswitch-2.13.7/lib/socket-util.h:51:6: note: declared here
   51 | bool inet_parse_active(const char *target, int default_port,
      |      ^~~~~~~~~~~~~~~~~
make[1]: *** [Makefile:2309: utilities/ovn-trace.o] Error 1
```

It's because ovn tag 20.12.0 doesn't match ovs tag 2.15.0, see [here](https://github.com/ovn-org/ovn/issues/128)

**3.**

the Dockerfile named as Dockerfile.dpdk1911, but actually it's 20.11

## Steps to Reproduce the Problem

1. make build-dpdk

## Additional Info

- Kubernetes version:

  **Output of `kubectl version`:**

Not a k8s install problem, but a dpdk build problem.

- kube-ovn version:
  
 ```base
 v1.12
```

- operation-system/kernel version:
  
  **Output of `awk -F '=' '/PRETTY_NAME/ { print $2 }' /etc/os-release`:**
  **Output of `uname -r`:**

 ```bash
Ubuntu 22.04 LTS 
5.15.0-25-generic

CentOS Linux 8
4.18.0-348.7.1.el8_5.x86_64
 ```

### PR DESCRIPTION
**Pull request by singchia**
# Pull Request

- [x] Make sure you have followed [Kube-OVN Code Style](https://github.com/kubeovn/kube-ovn/blob/master/CODE_STYLE.md).

## What type of this PR

Examples of user facing changes:

- Bug fixes
  - dpdk build failed because of base image centos8 reached End Of Life
  - ovn 21.03 tarball unmatch with ovs 2.15 tarball, like the ```inet_parse_active ``` function.
  - renaming to right name of image

## Which issue(s) this PR fixes

Fixes #3545

## WHAT

copilot:summary

copilot:poem

## HOW

copilot:walkthrough

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -111,7 +111,7 @@ build-debug: build-go
 
 .PHONY: build-dpdk
 build-dpdk:
-	docker buildx build --platform linux/amd64 -t $(REGISTRY)/kube-ovn-dpdk:19.11-$(RELEASE_TAG) -o type=docker -f dist/images/Dockerfile.dpdk1911 dist/images/
+	docker buildx build --platform linux/amd64 -t $(REGISTRY)/kube-ovn-dpdk:19.11-$(RELEASE_TAG) -o type=docker -f dist/images/Dockerfile.dpdk2011 dist/images/
```
Comment (singchia) [inline Makefile:114]: the tag should also be changed, but it relates stuff in install.sh, so I left it, for now.

**Comment by oilbeater** [comment]
@singchia Thanks for your contribution to the DPDK stuff. It hasn't been actively maintained for a while, your efforts are particularly appreciated!

**Review by oilbeater — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (oilbeater) [reply Makefile:114]: I will deal with the naming issue later

**Comment by singchia** [comment]
with pleasure.

**Pull request merged**